### PR TITLE
chore(build): exclude Jasmine matchers from build results

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**

### CODE REVIEW

**Comment by Foxandxss** [comment]
The typings in that folder are excluded?

**Comment by pkozlowski-opensource** [comment]
> The typings in that folder are excluded?

? Not sure I got the question?

**Comment by Foxandxss** [comment]
There is a .d.ts in `/util` that I am not sure if that is bundled too.

**Comment by wesleycho** [comment]
That typing is just for our tests. It is not needed for exposure

File: gulpfile.js [hunk 1]
```diff
@@ -41,7 +42,9 @@ var esmProject = ts.createProject('tsconfig-es2015.json');
 gulp.task('clean:build', function() { return del('dist/'); });
 
 gulp.task('cjs', function() {
-  var tsResult = gulp.src([PATHS.src, PATHS.typings, '!' + PATHS.specs]).pipe(sourcemaps.init()).pipe(ts(csjProject));
+  var tsResult = gulp.src([PATHS.src, PATHS.typings, '!' + PATHS.specs, '!' + PATHS.testHelpers])
+                     .pipe(sourcemaps.init())
+                     .pipe(ts(csjProject));
```
Comment (wesleycho) [inline gulpfile.js]: Typo

Comment (pkozlowski-opensource) [inline gulpfile.js]: Not part of my change though :-) This is part of the diff only due to clang....

Comment (wesleycho) [inline gulpfile.js]: I know, but it's an easy fix a few lines up :)

**Pull request closed without merging**